project/BalboaAdmin.scala

```diff
@@ -5,6 +5,7 @@ import scoverage.ScoverageSbtPlugin
 
 object BalboaAdmin {
   lazy val settings: Seq[Setting[_]] = BuildSettings.projectSettings ++ Seq(
+    scalaVersion := "2.11.8",
```

Comment:
Since many of these projects take their settings from BuildSettings, are you sure you need scalaVersion in each of these? It doesn't compile to scala 2.11 unless you put the setting in each sub project?